core/web/transfer_controller.go

```diff
@@ -33,7 +33,7 @@ func (tc *TransfersController) Create(c *gin.Context) {
 		jsonAPIError(c, http.StatusBadRequest, err)
 	} else if tx, err := store.TxManager.CreateTxWithEth(from, tr.DestinationAddress, tr.Amount); err != nil {
 		jsonAPIError(c, http.StatusBadRequest, fmt.Errorf("Transaction failed: %v", err))
-	} else if txp, err := presenters.NewTx(tx); err != nil {
+	} else if txp := presenters.NewTx(tx); false {
 		jsonAPIError(c, http.StatusInternalServerError, err)
 	} else {
 		jsonAPIResponse(c, txp, "transaction")
```

Comment:
Because the above scenario is impossible, you can collapse this even further. This:

```go
	} else if txp := presenters.NewTx(tx); false {
		jsonAPIError(c, http.StatusInternalServerError, err)
	} else {
		jsonAPIResponse(c, txp, "transaction")
```

can simply become:

```go
	} else {
		jsonAPIResponse(c, presenters.NewTx(tx), "transaction")
```

All for less lines of code 🍾